stubs/SQLAlchemy/sqlalchemy/sql/selectable.pyi

```diff
@@ -191,7 +191,7 @@ class Values(Generative, FromClause):
     name: Any
     literal_binds: Any
     def __init__(self, *columns, **kw) -> None: ...
-    def alias(self: Self, name: Any | None, **kw) -> Self: ...  # type: ignore[override]
+    def alias(self: Self, name: Incomplete | None, **kw) -> Self: ...  # type: ignore[override]
```

Comment:
None of the ones in this file are stubgen artefacts; they were all added in https://github.com/python/typeshed/pull/7603/files. _But_, there's no source-code comment indicating why they used `Any` here (and the comments on the PR thread give no insight), so I think it's fine to change the `Any` uses in this file to `Incomplete`.